### ISSUE TITLE
Use with Apollo's graphiql?

### ISSUE THREAD

**Issue by KyleAMathews** [OneGraph/graphiql-explorer#3]
Can this be used with Apollo's version of graphiql? Gatsby is thinking about switching over to it for the tabs.

**Comment by dwwoelfel** [comment]
Are they using https://github.com/prisma/graphql-playground? 

If so, it should be able to slot in nicely, since they're already using GraphiQL components. We'll need to support custom styles or somehow inherit the styles they're using.

**Comment by schickling** [comment]
Yes they are. @huvik can help out here. Excited to see the Playground be part of OneGraph 🚀

**Comment by KyleAMathews** [comment]
Yikes, yeah meant Prisma not Apollo 😅

**Comment by wtrocki** [comment]
graphql-playground quickly gaining a lots of popularity so having explorer integrated. 
I guess first integration do not need to look perfect as community can style that later. 
It's more about capabilities really.

**Comment by Huvik** [comment]
This looks really nice 🚀, I think it would nicely fit next to documentation and schema drawer on the right side of the playground. 

Regarding styling, there should be better documentation. I will try to improve it in the coming weeks so its easier to grasp.

**Comment by FluorescentHallucinogen** [comment]
What about Explorer plugin for [Altair](https://github.com/imolorhe/altair)?

See this feature request in Altair repo: https://github.com/imolorhe/altair/issues/534.
